Framework/PythonInterface/test/python/mantid/dataobjects/EventListTest.py

```diff
@@ -41,6 +47,9 @@ def test_event_list_addWeightedEventQuickly(self):
         self.assertEqual(el.getEventType(), EventType.WEIGHTED)
         self.assertEqual(el.getTofs()[0], float(0.123))
         self.assertEqual(el.getPulseTimes()[0], DateAndTime(42))
+        self.assertEqual(el.getPulseTimesAsNumpy()[0], gps_epoch_plus_42_nanoseconds)
+        self.assertEqual(el.getPulseTimesInNanosecondsFromUnixEpoch()[0],
+                         gps_epoch_plus_42_nanoseconds.astype('int64'))
```

Comment:
Why unsigned above, but signed here?

In general, is there an actual need for using `unsigned`? It has too many pitfalls, so unless required we may want to try and avoid it?

Reply:
Looking at https://docs.scipy.org/doc/numpy/reference/arrays.datetime.html#datetime-units, it seems numpy is actually using *signed*.